exonum-java-binding/common/src/main/java/com/exonum/binding/common/blockchain/Block.java

```diff
@@ -71,14 +68,11 @@
 
   /**
    * Root hash of the Merkle tree of transactions in this block.
-   * These transactions can be accessed with {@link Blockchain#getBlockTransactions(Block)}.
```

Comment:
I'd not just remove these — that's a regression — but think how the docs can be improved.